pkgs/os-specific/linux/tuna/default.nix

```diff
@@ -16,13 +16,12 @@ buildPythonApplication rec {
   pname = "tuna";
   version = "0.15";
 
-  src = fetchgit {
-    url = "https://git.kernel.org/pub/scm/utils/${pname}/${pname}.git";
-    rev = "v${version}";
-    sha256 = "sha256-lRHlbdCQ0NcjcWgLvCze67kN8NsK0f5RmKfPbkHhk78=";
+  src = fetchzip {
+    url = "https://git.kernel.org/pub/scm/utils/tuna/tuna.git/snapshot/tuna-v0.15.tar.gz";
```

Comment:
Am I missing something or why are we not substituting the version number here?

Reply:
Ah, that's a mistake, thanks.

Reply:
Fixed.